lib/trento/software_updates/policy.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Trento.SoftwareUpdates.Policy do
+  @moduledoc """
+  Policy for the SoftwareUpdates resource
+
+  User with the ability all:all can edit suma settings and clear suma settings.
+  User with the ability all:suma_settings can edit suma settings and clear suma settings.
+  """
+
+  @behaviour Bodyguard.Policy
+
+  import Trento.Support.PolicyHelper
+  alias Trento.SoftwareUpdates
+  alias Trento.Users.User
+
+  def authorize(action, %User{} = user, SoftwareUpdates)
```

Comment:
The 3rd argument should be `Trento.SoftwareUpdates.Settings`.
Remember that we are protecting the settings, not the updates entries